source/install/install-rhel-66-mattermost.rst

```diff
@@ -0,0 +1,86 @@
+..  _install-rhel-66-mattermost:
+
+Installing Mattermost Server
+============================
+
+Install Mattermost Server on a 64-bit machine.
+
+Assume that the IP address of this server is 10.10.10.2
+
+**To install Mattermost Server on RHEL 6.6**
+
+1. Log into the server that will host Mattermost Server and open a terminal window.
+
+2. Download `the latest version of the Mattermost Server <https://docs.mattermost.com/administration/upgrade.html#version-archive>`_. In the following command, replace ``X.X.X`` with the version that you want to download:
+
+  ``wget https://releases.mattermost.com/X.X.X/mattermost-X.X.X-linux-amd64.tar.gz``
+
+3. Extract the Mattermost Server files.
+
+  ``tar -xvzf *.gz``
+
+4. Move the extracted file to the ``/opt`` directory.
+
+  ``sudo mv mattermost /opt``
+
+5. Create the storage directory for files.
+
+  ``sudo mkdir /opt/mattermost/data``
+
+  .. note::
+    The storage directory will contain all the files and images that your users post to Mattermost, so you need to make sure that the drive is large enough to hold the anticipated number of uploaded files and images.
+
+6. Set up a system user and group called ``mattermost`` that will run this service, and set the ownership and permissions.
+
+  a. ``sudo useradd --system --user-group mattermost``
+  b. ``sudo chown -R mattermost:mattermost /opt/mattermost``
+  c. ``sudo chmod -R g+w /opt/mattermost``
+
+7. Set up the database driver in the file ``/opt/mattermost/config/config.json``. Open the file as root in a text editor and make the following changes:
+
+  -  If you are using PostgreSQL:    
+    1.  Set ``"DriverName"`` to ``"postgres"``
+    2.  Set ``"DataSource"`` to the following value, replacing ``<mmuser-password>``  and ``<host-name-or-IP>`` with the appropriate values:
+     ``"postgres://mmuser:<mmuser-password>@<host-name-or-IP>:5432/mattermost?sslmode=disable&connect_timeout=10"``.
+  -  If you are using MySQL:    
+    1.  Set ``DriverName"`` to ``"mysql"``
+    2.  Set ``"DataSource"`` to the following value, replacing ``<mmuser-password>``  and ``<host-name-or-IP>`` with the appropriate values:
+      ``"mmuser:<mmuser-password>@tcp(<host-name-or-IP>:3306)/mattermost?charset=utf8"``
+
+8. Test the Mattermost server to make sure everything works.
+
+    a. Change to the ``bin`` directory:
+      ``cd /opt/mattermost/bin``
+    b. Start the Mattermost server as the user mattermost:
+      ``sudo -u mattermost /.platform``
+
+  When the server starts, it shows some log information and the text ``Server is listening on :8065``. You can stop the server by typing ``CTRL-C`` in the terminal window.
```

Comment:
Previously we've gone with CTRL+C for keyboard shortcuts, i.e. "+" between buttons and no monospace highlighting.

Wondering your thoughts on which convention is better? Should we add a style guide on which to use?